setup.py

```diff
@@ -51,4 +51,5 @@ def is_release():
     version=version,
     description=description,
     long_description=long_description,
+    packages=find_packages(exclude=("flask-dapr",))
```

Comment:
can you specify this option inside setup.cfg ? I would like to minimize the setting inside setup.py.